### PR TITLE
0060 JS-Simsport

### PR DESCRIPTION
**Pull request by Jordessey**
Hi There.

I am an enthusiast maker that is looking to create low cost hardware to allow people to build their dream sim racing/farming/flight rig without spending thousands of dollars.

### CODE REVIEW

File: 1209/0060/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Exapandable Dash
+owner: JS SimSport
```
Comment (tannewt) [inline 1209/0060/index.md:4]:
```suggestion
owner: JS-SimSport

```

**Review by tannewt — CHANGES_REQUESTED** [review]
hardware is easyeda json under MIT. Software is CPP arduino under MIT. One edit to get the CI test happy.

**Review by Jordessey — COMMENTED** [review]
Fixing Owner/Title

**Review by Jordessey — COMMENTED** [review]
Foxing Owner

**Review by tannewt — APPROVED** [review]

**Pull request merged**